lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -124,12 +104,13 @@ public function register_routes() {
 	 *
 	 * @since 6.5.0
 	 *
+	 * @param WP_REST_Request $request Full details about the request.
 	 * @return true|WP_Error True if the request has read access, WP_Error object otherwise.
 	 */
-	public function get_font_faces_permissions_check() {
+	public function get_items_permissions_check( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable -- required by parent class
 		$post_type = get_post_type_object( $this->post_type );
 
-		if ( ! current_user_can( $post_type->cap->edit_posts ) ) {
+		if ( ! current_user_can( $post_type->cap->read ) ) {
 			return new WP_Error(
```

Comment:
Here checking `cap->read` rather than `cap->edit_posts`. In reality they all map to `edit_theme_options`, but checking for read capability seems more correct when getting items.